de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/ExporterImporterWizardTest.java

```diff
@@ -108,28 +98,51 @@ public void testHTMLExportFEA() {
 		assertTrue("Index file has been successfully created.", htmlExportIndexFile.exists());
 		assertTrue("ConfigurationTree file has been successfully created.", htmlExportCTFile.exists());
 		assertTrue("ElementConfiguration file has been successfully created.", htmlExportECFile.exists());
+	}
+	
+	@Test
+	public void testMatExportImport() {
+		Concept conceptMass = ConceptXmiLoader.loadConceptFromPlugin(de.dlr.sc.virsat.model.extension.budget.mass.Activator.getPluginId() + "/concept/concept.xmi");	
+		SWTBotTreeItem massEquipment = addElement(MassEquipment.class, conceptMass, ec);
+		openEditor(massEquipment);
 		
-		// Workaround to shells sometimes not being valid anymore after closing a wizard
-		// See https://wiki.eclipse.org/SWTBot/Troubleshooting#WidgetNotFoundException_when_stepping_through_SWTBot_test_in_Eclipse_debugger for details
-		bot.shell().activate();
+		final String OLD_VALUE = "45.0";
+		final String NEW_VALUE = "30.0";
+		
+		renameField(MassEquipment.PROPERTY_MASS, OLD_VALUE);
+		
+		openVirSatExporter("Mat Export Wizard");
+		
+		// Configure the export
+		bot.tree().expandNode(EC_PATH).select();
+		Path matExportFilePath = exportFolderPath.resolve("export.mat");
+		bot.comboBox().setText(matExportFilePath.toString());
+		finishWizard();
+		
+		// Assert that we correctly exported a file
+		assertTrue("A file has been successfully created.", matExportFilePath.toFile().exists());
+		
+		// Cause a change
+		renameField(MassEquipment.PROPERTY_MASS, NEW_VALUE);
 
+		openVirSatImporter("Mat Import Wizard");
```

Comment:
Can we maybe use the String from the Wizard definition? Looks like "magic text" here :D

Reply:
That would be cool, but sadly the string is in the plugin.xml of the ui.engine plugin.
I tried using our ResourceBuilder hoping that it would maybe produce a PluginXML.java where the String is written into but apparantly that doesnt happen.